files/templates/scrapeconfigs/additional-scrape-configs.template.yaml

```diff
@@ -517,6 +517,20 @@
   - source_labels: [__name__]
     regex: (nginx_ingress_controller_bytes_sent_bucket|nginx_ingress_controller_request_size_bucket|nginx_ingress_controller_response_duration_seconds_bucket|nginx_ingress_controller_response_size_bucket)
     action: drop
+  # drop uid label from kube-state-metrics
+  - source_labels: [app,uid]
+    separator: ;
+    regex: kube-state-metrics;(.+)
+    replacement: ""
+    action: replace
```

Comment:
You prefer using `replace` than `drop` in case those labels are used somewhere ?

Reply:
Well i want to remove those labels from ksm only, Hence thé 2 source labels. If i were to use drop, it would also drop the app label 😅

Reply:
Sure but I think labeldrop would also drop the app label right? Maybe you Can fix it tomorrow, but it's useful to add it to the prometheus-operator pr to enable ksm

Reply:
Added here but needs to be tested

Reply:
I love that :-)